src/mpd.c

```diff
@@ -85,7 +80,40 @@ static bool mpd_plugin_httpd;
 static char *default_pl_dir;
 static bool allow_modifying_stored_playlists;
 
-#define COMMAND_ARGV_MAX 37
+
+/**
+ * from MPD source:
+ * *
+ * * The most we ever use is for search/find, and that limits it to the
+ * * number of tags we can have.  Add one for the command, and one extra
+ * * to catch errors clients may send us
+ * *
+ *  static constexpr std::size_t COMMAND_ARGV_MAX = 2 + TAG_NUM_OF_ITEM_TYPES * 2;
+ *
+ * https://github.com/MusicPlayerDaemon/MPD/blob/master/src/command/AllCommands.cxx
+ */
+#define COMMAND_ARGV_MAX 70
+
+/**
+ * config:
+ * max_command_list_size KBYTES
+ * The maximum size a command list. Default is 2048 (2 MiB).
+ *
+ * https://github.com/MusicPlayerDaemon/MPD/blob/master/src/client/Config.cxx
+ */
+#define CLIENT_MAX_COMMAND_LIST_DEFAULT		(2048*1024)
+
+static struct {
```

Comment:
this is an anonymous struct for a single member, is it expected to contain more in the future?  How about the current static vars such as allow_modifying_stored_playlists, shouldn't those be in this structure too (as a separate commit)?

Also, probably better to declare this structure named, and then define mpd_config as static variable.

(Config is very generic easy to clash, and IMO inconsistent with the rest of the file